pandas/tests/frame/test_alter_axes.py

```diff
@@ -1391,14 +1417,6 @@ def test_set_index_preserve_categorical_dtype(self):
             result = result.reindex(columns=df.columns)
             tm.assert_frame_equal(result, df)
 
-    def test_ambiguous_warns(self):
```

Comment:
These use cases are now covered by the `test_rename_positional_raises` test case above, so deleted as duplicative